src/merkle/leaf.iced

```diff
@@ -9,6 +9,20 @@ exports.ChainTail = class ChainTail
 
 #--------------------------
 
+exports.ResetChainTail = class ResetChainTail
+  constructor : ({@seqno, @payload_hash}) ->
+  to_json : () -> [ @seqno, @payload_hash ]
+
+#--------------------------
+
+is_positive_int = (x) ->
+  return false unless typeof(x) is 'number'
+  return false unless Math.floor(x) is x
+  return false if x < 0
+  return true
```

Comment:
Can we use the `Number.isInteger` function? Or maybe [the polyfill suggested](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) in that function's docs?

Reply:
👍